vagrant/provisioning/roles/common/tasks/main.yml

```diff
@@ -97,7 +97,7 @@
     line: "{{ item }}"
   loop:
     - "{{ arkcase_host_address | default('127.0.0.1') }} arkcase-host"
-    - "{{ ansible_default_ipv4.address }} {{ internal_host }}"
+    - "{{ ansible_default_ipv4.address | default(ansible_all_ipv4_addresses[0]) }} {{ internal_host }}"
```

Comment:
Interesting that `ansible_default_ipv4.address` is not identical to `ansible_all_ipv4_addresses[0]` ... good catch here.